### PR TITLE
Compass to inquiry pr

### PR DESCRIPTION
**Pull request by pavankumar0408**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by pavankumar0408** [comment]
Added changes for server side evaluation based on eval{mode} property from the questionset. If mode is 'server' test evaluation will happen at server side and if mode is 'client' test evaluation will happen at client side (which is currently existing flow).

**Comment by rajnishdargan** [comment]
@pavankumar0408 
Please build the web-component using command `npm run build-web-component` and update the npm package version.
For reference: https://github.com/Sunbird-inQuiry/player/pull/44/files

If you have the JIRA ticket kindly update the PR title to below format:
`Issue #IQ-xxx feat: Proper message for PR header or the Jira ticket title`

**Comment by pavankumar0408** [comment]
@rajnishdargan @sajeshkayyath, required access for adding reviewers

File: projects/quml-library/src/lib/quml-library.service.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ export class QumlLibraryService {
 
   constructor(public utilService: UtilService) { }
 
-  async initializeTelemetry(config: QumlPlayerConfig, parentConfig: IParentConfig) {
+  initializeTelemetry(config: QumlPlayerConfig, parentConfig: IParentConfig) {
```
Comment (rajnishdargan) [inline projects/quml-library/src/lib/quml-library.service.ts:27]: @pavankumar0408 Was this async await not needed here?

**Review by rajnishdargan — APPROVED** [review]

**Review by rajnishdargan — APPROVED** [review]

**Review by sajeshkayyath — APPROVED** [review]

**Pull request merged**